src/editor/actions/actions.js

```diff
@@ -204,6 +204,14 @@ export function updateEditorCursor(line, col, forceUpdate = false) {
   return { type: "UPDATE_CURSOR", line, col, forceUpdate };
 }
 
+export function moveToNextChunk() {
```

Comment:
there is already an action `moveCursorToNextChunk`

Reply:
oh, thanks for the heads up. Of course there is! :|